app/Http/Controllers/Api/V1/IndependentActivityController.php

```diff
@@ -761,6 +765,18 @@ public function searchPreview(Organization $suborganization, IndependentActivity
         ], 200);
     }
 
+    /**
+     * Download XApi File
+     *
```

Comment:
Need to add docs as per following https://beyondco.de/docs/laravel-apidoc-generator/getting-started/documenting-your-api